res/css/photon-components.css

```diff
@@ -121,3 +154,139 @@
 .photon-message-bar-error .photon-button:hover:active {
   background-color: var(--red-90);
 }
+
+/* Label */
+
+.photon-label,
+.photon-label-micro {
+  display: inline-flex;
+  color: var(--grey-90);
+}
+
+.photon-label {
+  padding: 3px 0;
+  font-size: 13px;
+}
+
+.photon-label-micro {
```

Comment:
I know you find that the default photon styling is too big for our app, but I do think that in our app everything is too small, and especially the text. I think we should have bigger sizes by default, then the user could always zoom out from the browser.

This discussion is out of scope for this PR and I'm OK merging a "micro" flavor for now. But I'm not happy with our default sizes and I think we should discuss it later (maybe next all hands ?).